src/test/java/hudson/tasks/_ant/AntTargetAnnotationTest.java

```diff
@@ -23,7 +23,8 @@
 public class AntTargetAnnotationTest {
     
     @Rule
-    public JenkinsRule r = new JenkinsRule();
```

Comment:
I have actually been preferring the name `r`, since `j` is too easily confused with the `Jenkins` instance itself.

Reply:
My concern here was essentially because the same name is used for `RestartableJenkinsRule r`. No worries I revert them.

Reply:
My style (FWIW!) is to use `RestartableJenkinsRule rr`.